joblib/hashing.py

```diff
@@ -77,6 +78,32 @@ def save(self, obj):
                 obj = _MyHash(func_name, inst, cls)
         Pickler.save(self, obj)
 
+    def memoize(self, obj):
+        """Store an object in the memo."""
+
+        # The Pickler memo is a dictionary mapping object ids to 2-tuples
+        # that contain the Unpickler memo key and the object being memoized.
+        # The memo key is written to the pickle and will become
+        # the key in the Unpickler's memo.  The object is stored in the
+        # Pickler memo so that transient objects are kept alive during
+        # pickling.
+
+        # The use of the Unpickler memo length as the memo key is just a
+        # convention.  The only requirement is that the memo values be unique.
+        # But there appears no advantage to any other scheme, and this
+        # scheme allows the Unpickler memo to be implemented as a plain (but
+        # growable) array, indexed by memo key.
+        if self.fast:
+            return
+        # Separate bytes and unicode shared by objects in the pickle graph,
+        # to avoid cache invalidation when unpickling a subobject of the graph
```

Comment:
I think that you should state in the comment above that this is the only place where this method differs from the parent's method.